AutomatedTesting/Gem/PythonTests/editor/EditorScripts/EditorWorkflow_EditorCameraBeThisCameraIsClearedWhenChangingLevel.py

```diff
@@ -0,0 +1,159 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+
+class Tests:
+    first_level_created = (
+        "Successfully created first level",
+        "Failed to create first level"
+    )
+    second_level_created = (
+        "Successfully created second level",
+        "Failed to create second level"
+    )
+    editor_camera_initial_position_identical_in_first_and_second_level = (
+        "Initial camera positions match in first and second level",
+        "Initial camera positions do not match in first and second level",
+    )
+    entity_position_changed_when_not_in_be_this_camera = (
+        "Entity position did not change when setting view position while not in 'Be this camera'",
+        "Entity position did change when setting view position while not in 'Be this camera'"
+    )
+    view_position_changed_when_moving_entity_when_not_in_be_this_camera = (
+        "View position did not change when moving entity while not in 'Be this camera' mode",
+        "View position did change when moving entity while not in 'Be this camera' mode",
```

Comment:
Suggestion: These could be written in a way where the failure case is more obviously a failure, possibly even using the word "failed". For example:
"Initial camera positions failed to match in first and second level"
"Entity position failed to remain the same when setting view position..."
"View position failed to remain the same when moving entity..."

It would also be nice if the success case avoided using negatives to make it even more obvious which statement represents success and which represents failure. For example:
"Entity position remained the same when setting view position outside of 'Be this camera' mode"
"View position remained then same when moving entity outside of 'Be this camera' mode"

Reply:
Hmm I mean you will get a big red failure output anyway right? I think it's obvious that something has gone wrong when the test fails (the condition isn't met), adding another 'failed' to the message isn't going to make a great deal of difference in my mind.

I'll have a look if I can simplify the wording slightly but again I think the test failing and the context of both messages when debugging the test is sufficient.